src/components/Api.js

```diff
@@ -0,0 +1,175 @@
+export default class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl;
+    this.headers = options.headers;
+  }
+
+  getInitialCards() {
+    return fetch(this.baseUrl + "/cards", {
+      headers: {
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)